app/presenters/aggregated_tag_mapping_presenter.rb

```diff
@@ -0,0 +1,20 @@
+class AggregatedTagMappingPresenter < SimpleDelegator
+  def errored?
+    presented_tag_mappings.any? { |tag_mapping| tag_mapping.state == 'errored' }
+  end
+
+  def error_messages
+    presented_tag_mappings.flat_map do |tag_mapping|
+      next unless tag_mapping.message
+
+      # splits message on ".", but keeps it in the message.
+      tag_mapping.message.split(/(?<=[.])/)
```

Comment:
We no longer split messages because they are now an array in the database. Please rebase so you can see that change (more details [here](https://github.com/alphagov/content-tagger/blob/master/app/models/tag_mapping.rb#L12))